### PR TITLE
remove creation of openshift-infra namespace

### PR DESCRIPTION
**Pull request by atiratree**
There should be only one source of truth for creation of openshift-infra namespace. This is introduced in https://github.com/openshift/cluster-kube-controller-manager-operator/pull/570

### CODE REVIEW

**Comment by atiratree** [comment]
KCM-o change is in
/hold cancel

**Review by soltysh — APPROVED** [review]
/lgtm
/retest-required

**Comment by soltysh** [comment]
/assign @sttts 
for approval

File: pkg/cmd/openshift-apiserver/openshiftapiserver/openshift_apiserver.go [hunk 1]
```diff
@@ -519,12 +519,6 @@ func (c *completedConfig) EnsureOpenShiftInfraNamespace(_ genericapiserver.PostS
 		return err
 	}
 
-	_, err = coreClient.Namespaces().Create(context.TODO(), &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: namespaceName}}, metav1.CreateOptions{})
-	if err != nil && !kapierror.IsAlreadyExists(err) {
-		utilruntime.HandleError(fmt.Errorf("error creating namespace %q: %v", namespaceName, err))
-		return err
-	}
-
```
Comment (sttts) [inline pkg/cmd/openshift-apiserver/openshiftapiserver/openshift_apiserver.go:527]: won't run into the error below?

Comment (atiratree) [reply pkg/cmd/openshift-apiserver/openshiftapiserver/openshift_apiserver.go:527]: openshift-infra is one of the first things we create in bootkube phase now so it should be okay https://github.com/openshift/cluster-kube-controller-manager-operator/blob/master/bindata/bootkube/manifests/0000_00_namespace-openshift-infra.yaml

Comment (sttts) [reply pkg/cmd/openshift-apiserver/openshiftapiserver/openshift_apiserver.go:527]: ack

**Pull request merged**